sentry/src/db/campaign.rs

```diff
@@ -603,8 +607,14 @@ mod test {
 
         let campaign = DUMMY_CAMPAIGN.clone();
 
+        let channel_chain = app
+            .config
+            .find_chain_of(DUMMY_CAMPAIGN.channel.token)
+            .expect("Channel token should be whitelisted in config!");
+        let channel_context = channel_chain.with_channel(DUMMY_CAMPAIGN.channel);
```

Comment:
I would suggest doing the same thing here as well - use `GANACHE_CONFIG` instead of `setup_dummy_app`